arch/arm/core/aarch64/isr_wrapper.S

```diff
@@ -45,6 +46,11 @@ SECTION_FUNC(TEXT, _isr_wrapper)
 #else
 	bl	z_soc_irq_get_active
 #endif /* !CONFIG_ARM_CUSTOM_INTERRUPT_CONTROLLER */
+
+	/* Spurious */
+	cmp	x0, #(GIC_INTID_SPURIOUS)
+	beq	spurious_continue
```

Comment:
**Note for future reference**
This is GIC-specific, so it should have gone inside the `#if !defined(CONFIG_ARM_CUSTOM_INTERRUPT_CONTROLLER)` block.

Alternatively, the interrupt controller `get_active` interface (i.e. `arm_gic_get_active` and `z_soc_irq_get_active`) should have been abstracted to return an interrupt controller-agnostic spurious interrupt number so that the arch code can process it without interrupt controller-specific knowledge.